packages/devtools/.eslintrc.js

```diff
@@ -0,0 +1,186 @@
+const rules = {
+  'arrow-body-style': 'error',
+  'arrow-parens': ['off', 'as-needed'],
+  camelcase: 'error',
+  'capitalized-comments': 'error',
```

Comment:
I'd suggest either switching this off completely (probably my preference), or using the [ignore consecutive comments](https://eslint.org/docs/rules/capitalized-comments#ignoreconsecutivecomments) option.